docs/installation_guide.rst

```diff
@@ -166,6 +166,49 @@ At this point the rotki docker container should be running and you
 should be able to access rotki frontend, open your browser and go to
 it at ``http://localhost:8084``. You should be able to see the rotki login screen.
 
+
+Configuring backend in docker
+-------------------------------
+It is possible to change the configuration of the backend in the docker container. You can do so
+using two different approaches.
+
+The first approach include mounting a config volume::
+
+   docker run -d --name rotki \
+       -p 8084:80 \
+       -v $HOME/.rotki/data:/data \
+       -v $HOME/.rotki/logs:/logs \
+       -v $HOME/.rotki/config:/config \
+       rotki/rotki:latest
+
+You need to create a ``rotki_config.json`` file and put it inside ``$HOME/.rotki/config`` directory.
+
+.. code-block:: javascript
+
+    {
+       "loglevel": "info",
+       "logfromothermodules": true,
+       "sleep-secs": 22,
+       "max_size_in_mb_all_logs": 550,
+       "max_logfiles_num": 3
```

Comment:
You mix here three name formats. Is better to stick with one. For python variables also is better `log_from_other_modules` in the python code

Reply:
I am aware but the `rotki_config.json` has had that format for a long time now.

Reply:
This is not something that was introduced right now, I just reused the existing settings format.